src/smc-webapp/info/links.tsx

```diff
@@ -108,8 +109,12 @@ export const CONNECT_LINKS = {
   discord: {
     bold: true,
     icon: "fab fa-discord",
-    href: "https://discord.gg/Pz3h5bH",
-    link: "Discord - chat about CoCalc"
+    href: DISCORD_INVITE,
+    link: (
+      <span>
+        Discord - chat about <SiteName />
```

Comment:
I disagree with this change you made.  Anybody who changes the site name is NOT changing the discord link... and the discord link is definitely to CoCalc's discord chat.